features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -284,6 +475,119 @@ Scenario: Supplier changes their answers before submission
       | Sip quietly                         |                             |
       | Provide biscuits                    |  Only the finest            |
 
+@skip-staging
+Scenario: Supplier changes their answers before submission
+  Given that supplier has applied to be on that framework
+  And we accept that suppliers application to the framework
+  And that supplier returns a signed framework agreement for the framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
+  And that supplier has filled in their response to that brief but not submitted it
+  And I go to that brief page
+  And I click 'Apply for this opportunity'
+  Then I am on 'Before you start' page
+
+  When I click 'Continue application'
+  Then I am on 'When is the earliest the specialist can start work?' page
+  And I see 'The buyer needs the specialist to start: Saturday 31 December 2016' replayed in the question advice
```

Comment:
I wonder if we should bring these dates forward a bit so the tests feel fresh and relevant to now... Probably not worth the hassle though!